doc/gov/chapters/chapter02.md

```diff
@@ -24,34 +24,64 @@ As an open-source community organization, CNTT has developed the following proce
 
 - **Guide To Getting Started**
 
-  - **How do I start the process of joining?** Onboarding starts as soon as you submit a request to join CNTT via email to xxxx@xxxxx.com.
+  - **How do I start the process of becoming a participant?** Onboarding starts as soon as you submit a request to join CNTT via email to cntt-gov@lists.opnfv.org. You will receive additional information such as links to the CNTT Getting Started wiki and additional onboarding materials.
```

Comment:
Maybe we should describe why one wants to be a participant? At the moment this is needed to be able to create pr-s. 

Most easy onboarding would be automatic.

Reply:
@CsatariGergely , Can you elaborate your comment? Or are you good with the changes I made? Thx, J